src/thorlabs_cube/driver/kcube/kpa.py

```diff
@@ -0,0 +1,400 @@
+import struct as st
+
+from thorlabs_cube.driver.base import _Cube
+from thorlabs_cube.driver.message import MGMSG, Message, MsgError
+
+
+_CHANNEL: int = 0x01
+
+
+class Kpa(_Cube):
```

Comment:
Why does this class not inherit from a T-Cube (e.g. [TPA101](https://www.thorlabs.com/catalogpages/obsolete/2016/TPA101.pdf)) like the other K-Cube controllers?

Reply:
Both TPA101 and KPA101 inherit the same methods and since TPA101 has been discontinued, it would be redundant as the KPA101 class can be used to technically run a TPA101 if needed.

Reply:
Didn't we discover in the manual that this is not the case? See pg. 27 of the [communication protocol](https://github.com/quantumion/thorlabs_cube/blob/master/doc/APT_Communications_Protocol_v38.pdf).